packages/preview/qooklet/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+    name = "qooklet"
+    version = "0.1.0"
+    compiler = "0.13.0"
+    repository = "https://github.com/ivaquero/qooklet.git"
+    entrypoint = "src/lib.typ"
+    authors = ["ivaquero"]
+    license = "MIT"
+    description = "A quick start template for scientific booklets"
```

Comment:
If this package is intended to be used a template, maybe it should have a `[template]` section in the manifest and a default file from which to start using it? If not, I think it would be better to change the description to something like "Utilities to typeset scientific booklets".

Reply:
I agree, I'll use utility instead of template.